lib/flag-filter.js

```diff
@@ -184,8 +184,12 @@ FlagFilter.prototype._transform = function (vm, enc, cb) {
 		vm.ptrname = vm.tags[DOCKER_PREFIX + INST_PTR_TAG];
 	}
 
-	/* And enable VMs owned by admin that have a service attached. */
-	if (vm.owner.login === 'admin' && vm.services.length > 0) {
+	/* 
+	 * And enable VMs owned by admin and poseidon that have a service 
```

Comment:
Can you change the comment to say:

> And enable VMs owned by admin or poseidon that have a service  attached.

(*or* instead of *and* to accurately reflect the code behavior)